src/Showcase/src/Smart.FA.Catalog.Showcase.Infrastructure/Data/Migrations/20220711112432_AddGivenBySmart.cs

```diff
@@ -0,0 +1,133 @@
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace Smart.FA.Catalog.Showcase.Infrastructure.Migrations
+{
+    public partial class AddGivenBySmart : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            // TrainingDetails.
+            migrationBuilder.Sql(@"
+               ALTER VIEW [Cfa].[v_TrainingDetails]
+               AS
+               SELECT training.Id
+                      ,details.Title
+                      ,details.Methodology
+                      ,details.Goal
+                      ,details.PracticalModalities
+                      ,topic.TopicId
+                      ,details.Language
+                      ,trainer.FirstName
+                      ,trainer.LastName
+                      ,trainer.Id AS TrainerId
+                      ,trainer.Title AS trainerTitle
+                      ,training.TrainingStatusTypeId
+                      ,trainer.ProfileImagePath
+                      ,training.IsGivenBySmart
+              FROM [Cfa].[Training] AS training
+                    INNER JOIN [Cfa].[TrainingLocalizedDetails] AS details
+                    ON training.Id = details.TrainingId
+                    INNER JOIN [Cfa].[TrainerAssignment] trainingTrainer
+                    ON training.Id = trainingTrainer.TrainingId
+                    INNER JOIN [Cfa].[Trainer] trainer
+                    ON trainingTrainer.TrainerId = trainer.Id
+                    INNER JOIN [Cfa].[TrainingTopic] topic
+                    ON training.Id = topic.TrainingId
+                    INNER JOIN Cfa.TrainingStatusType trainingStatusType
+                    ON trainingStatusType.Id = training.TrainingStatusTypeId
+                WHERE TrainingStatusType.Name = 'Published'");
+
+            // TrainingList.
+            migrationBuilder.Sql("DROP VIEW IF EXISTS [Cfa].[v_TrainingList]");
+            migrationBuilder.Sql(@"
+                CREATE VIEW [Cfa].[v_TrainingList]
```

Comment:
```suggestion
            migrationBuilder.Sql(@"
                ALTER VIEW [Cfa].[v_TrainingList]
```

I think at this point you can keep just the alter as normally the view should be already existing.